projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/ingest_to_oracle.py

```diff
@@ -18,12 +20,90 @@
 log = logging.getLogger(__name__)
 
 
+# Functions for escaping special characters
+def _is_plain_identifier(identifier: str) -> bool:
+    class InvalidOracleIdentifierError(BaseException):
+        pass
+
+    # Raise error identifier it starts with a number
+    if re.match("\\d", identifier):
+        raise InvalidOracleIdentifierError(
+            f"{identifier} is not a valid Oracle identifier. "
+            f"https://docs.oracle.com/en/error-help/db/ora-00904/"
+        )
+
+    # https://docs.oracle.com/en/error-help/db/ora-00904/
+    # Alphanumeric that doesn't start with a number
+    # Can contain and start with $, # and _
+    regex = "^[A-Za-z\\$#_][0-9A-Za-z\\$#_]*$"
+    return True if re.fullmatch(regex, identifier) else False
```

Comment:
nit: this could be simplified to 

```suggestion
    return bool(re.fullmatch(regex, identifier))
```